packages/diagrams/frontend/sirius-components-diagrams/src/renderer/helper-lines/useHelperLines.tsx

```diff
@@ -271,37 +297,42 @@ const getHelperLinesForResizeAndMove = (
 };
 
 export const useHelperLines = (): UseHelperLinesValue => {
-  const [enabled, setEnabled] = useState<boolean>(false);
+  const [enabled, setEnabled] = useState<boolean>(true);
```

Comment:
Are you sure about that change?

Reply:
Yes, like mention in the changelog, the helper lines should be activated by default from now

Reply:
I've separated this modification into a second commit